datasets/wili_2018/README.md

```diff
@@ -0,0 +1,25 @@
+Please refer to the WiLI paper for details about the dataset.
```

Comment:
can you add the yaml tags at the top of the dataset card ? you can find more info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Can you also use the dataset card template ? You can leave the sections contents with `[More Information Needed]` : https://github.com/huggingface/datasets/blob/master/templates/README.md

Reply:
@lhoestq I have updated the dataset card, I think it's fine now. Let me know if any further changes are required.